content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -0,0 +1,42 @@
+# Default disks and node configuration
+
+## Overview
+
+This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes **using Kubernetes label and annotation**, instead of using Longhorn API or UI.
+For more details, see [this enhancement doc](https://github.com/longhorn/longhorn/blob/v0.8.1/enhancements/20200319-default-disks-and-node-configuration.md).
```

Comment:
Not sure if we should refer to the design doc in the formal documentation.

Reply:
Can I say `If you are interested in the feature design, see here for details`

Reply:
The problem is we're not going to update the design doc, but we will always keep doc up to date after we add more features etc. So I prefer to not have a permanent link here to the design doc.

Of course, the design doc can help contributors to understand the feature better. But if the user asks for more details, we can also point them to the design doc. But that should be very rare if our official document serves its purpose well.

Reply:
OK. Then I will remove this link.